linux-packages/content/configure-tentacle.sh

```diff
@@ -100,6 +100,7 @@ function setupPollingTentacle {
     rolesstring=""
     workerpoolsstring=""
     machinetype=1
+    servercommsport=10943
```

Comment:
Out of curiosity, do we know why this was strictly only set to `10943` before?

Reply:
Probably because it is the default.